xds/src/main/java/io/grpc/xds/XdsLoadBalancer.java

```diff
@@ -150,32 +180,32 @@ private static String getPolicyNameOrNull(@Nullable LbConfig config) {
   @Override
   public void handleNameResolutionError(Status error) {
     if (xdsLbState != null) {
-      if (fallbackManager.fallbackBalancer != null) {
-        fallbackManager.fallbackBalancer.handleNameResolutionError(error);
-      } else {
-        xdsLbState.handleNameResolutionError(error);
-      }
+      xdsLbState.handleNameResolutionError(error);
+    }
+    if (fallbackManager.fallbackBalancer != null) {
+      fallbackManager.fallbackBalancer.handleNameResolutionError(error);
```

Comment:
Doesn't seem this path is covered in the unit test.
You probably want to examine the jacoco report to make sure all new code paths are covered.

Reply:
Done. The above lines of code was not covered either, although `xdsLbState.handleNameResolutionError` is not really implemented.